docs/app/Examples/modules/Dropdown/Types/Dropdown.js

```diff
@@ -1,31 +0,0 @@
-import React, { Component } from 'react'
-import { Dropdown } from 'stardust'
-
-export default class DropdownDropdownExample extends Component {
-  render() {
-    return (
-      <Dropdown text='File' action='hide'>
-        <Dropdown.Item text='New' />
-        <Dropdown.Item text='Open...' description='ctrl + o' />
-        <Dropdown.Item text='Save as...' description='ctrl + s' />
-        <Dropdown.Item text='Rename' description='ctrl + r' />
-        <Dropdown.Item text='Make a copy' />
-        <Dropdown.Item icon='trash' text='Move to trash' />
-        <Dropdown.Divider />
-        <Dropdown.Item text='Download As...' />
-        <Dropdown.Item text='Publish To Web'>
-          <Dropdown.Menu>
-            <Dropdown.Item text='Google Docs' />
-            <Dropdown.Item text='Google Drive' />
-            <Dropdown.Item text='Dropbox' />
-            <Dropdown.Item text='Adobe Creative Cloud' />
-            <Dropdown.Item text='Private FTP' />
-            <Dropdown.Item text='Another Service...' />
-          </Dropdown.Menu>
-        </Dropdown.Item>
-        {/* item text can also be defined as children */}
-        <Dropdown.Item>E-mail Collaborators</Dropdown.Item>
-      </Dropdown>
```

Comment:
We are not going to ship v1.0 with support for building custom Dropdown components in this way.  There is work yet to be done in the way of handling the `options` prop in conjunction with `Dropdown.Item` children.  Especially when it comes to nested menus and searching features.

Currently, we only support the `options` prop which gives a SSOT and makes searching a breeze.